python/py-kombine/Portfile

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-kombine
+version             0.8.3
+
+categories-append   science
+maintainers         {uoregon.edu:bfarr @bfarr} openmaintainer
+
+platforms           darwin
+license             GPL-3
+
+description         An embarrassingly parallel, kernel-density-based \
+                    ensemble sampler
+long_description    kombine is an ensemble sampler that uses a clustered \
+                    kernel-density-estimate proposal density, allowing for \
+                    massive parallelization and efficient sampling.
+homepage            https://github.com/bfarr/kombine
+
+master_sites        pypi:k/kombine
+distname            kombine-${version}
+checksums   rmd160  74c1ba9d7f4f1866f79f4628c868943afb9fabec \
+            sha256  da7a9542600c81df4b2ede2772b730dacef61ea3bc19b495f4fa445ff2b92f75 \
+            size    17495
+
+python.versions     27 34 35 36
```

Comment:
Someone can correct me if I'm wrong, but I think we're currently supporting 2.7 and the last two (not three) releases of python 3?

Reply:
@pmetzger,  it looks like py34 is (still) supported:

```
$ port search python34
...
python34 @3.4.8 (lang)
    An interpreted, object-oriented programming language
```

Reply:
It's still present. I'm not sure we're not trying to get things off of it. But anyway, it doesn't matter as this is harmless.

Reply:
For new python ports, please only support the latest version of 2 and 3. So py27 and py36 only unless there is some need to add an older version. Yes python34 still exists in Macports because it used to be the latest version of python 3 and there are still many ports that use it and have not been updated to 3.6.

Reply:
@mf2k Should new ports default to py27 or py36? (See the next comment in the series.)

Reply:
It depends on the python module. Mostly it should default to py36, but occasionally the py27 version still works better.

Reply:
@mf2k So this is set to default to py27 but I suspect it should be py36. I'll tag you below.